src/onegov/org/forms/document_form.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+from onegov.core.utils import normalize_for_url
```

Comment:
nit. future import spacing.
```suggestion
from __future__ import annotations

from onegov.core.utils import normalize_for_url
```